src/Trakx.CoinGecko.ApiClient/Clients/CoinGeckoClient_Public.cs

```diff
@@ -53,8 +52,8 @@ public CoinGeckoClient(
         string quoteCurrencyId = Constants.UsdCoin,
         CancellationToken cancellationToken = default)
     {
-        if(coinGeckoId.IsNullOrWhiteSpace()) throw new ArgumentException($"{nameof(coinGeckoId)} cannot be null or empty");
-        if(quoteCurrencyId.IsNullOrWhiteSpace()) throw new ArgumentException($"{nameof(quoteCurrencyId)} cannot be null or empty");
+        if (coinGeckoId.IsNullOrWhiteSpace()) throw new ArgumentException($"{nameof(coinGeckoId)} cannot be null or empty");
+        if (quoteCurrencyId.IsNullOrWhiteSpace()) throw new ArgumentException($"{nameof(quoteCurrencyId)} cannot be null or empty");
```

Comment:
Could this use `ArgumentException.ThrowIfNullOrWhiteSpace` instead?

Reply:
Good catch, thanks, done:
https://github.com/trakx/coingecko-api-client/pull/252/commits/c9a9f8b5d2633f4baa5ab7211be008d359086d9c